code/modules/mob/living/carbon/xenomorph/emote.dm

```diff
@@ -47,6 +47,10 @@
 	key = "hiss3"
 	sound = 'sound/voice/alien_hiss3.ogg'
 
+/datum/emote/living/carbon/xenomorph/kiss
+    key = "kiss"
+    key_third_person = "kisses"
+    message = "blows a kiss extending their claw outwards"
```

Comment:
```suggestion
    message = "blows a kiss."
```

Reply:
I'll change it again, but having a simple "Blows a kiss" is what I don't like about current emotes, far too bland and simple with no emotion or flavor.

Reply:
Ping me again on discord if you really dislike it, but for now that's what I want to move away from